internal/controller/utils_test.go

```diff
@@ -5,9 +5,12 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
 )
 
+var defaultUuid = uuid.MustParse("08ce96a3-3430-48a8-a3b2-b1c987a207ca")
```

Comment:
Some constants were defined in the user related test files but used in multiple other files: in this PR we moved them to where it makes sense.